pg_sparse/Cargo.toml

```diff
@@ -0,0 +1,32 @@
+[package]
+name = "pg_sparse"
+description = "Sparse vector similarity search in Postgres"
+version = "0.0.0"
+edition = "2021"
+license = "AGPL-3.0"
+
+[lib]
+crate-type = ["cdylib"]
+
+[features]
+default = ["pg15"]
+pg11 = ["pgrx/pg11", "pgrx-tests/pg11"]
+pg12 = ["pgrx/pg12", "pgrx-tests/pg12"]
+pg13 = ["pgrx/pg13", "pgrx-tests/pg13"]
+pg14 = ["pgrx/pg14", "pgrx-tests/pg14"]
+pg15 = ["pgrx/pg15", "pgrx-tests/pg15"]
+pg_test = []
+
```

Comment:
Can you add telemetry support to this extension, similar to how we did it in here https://github.com/paradedb/paradedb/pull/427